src/pages/Organize/License/Create/index.jsx

```diff
@@ -0,0 +1,202 @@
+/*
+ Copyright (C) 2021 Shruti Agarwal (mail2shruti.ag@gmail.com)
+
+ SPDX-License-Identifier: GPL-2.0
+
+ This program is free software; you can redistribute it and/or
+ modify it under the terms of the GNU General Public License
+ version 2 as published by the Free Software Foundation.
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License along
+ with this program; if not, write to the Free Software Foundation, Inc.,
+ 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+*/
+
+import React, { useState } from "react";
+
+// Widgets
+import { Alert, Button, InputContainer, Spinner } from "components/Widgets";
+
+// Required functions for calling APIs
+import { createCandidateLicense } from "services/licenses";
+
+const AddCandidateLicense = () => {
+  const options = [
+    {
+      id: 1,
+      value: 1,
+    },
+    {
+      id: 2,
+      value: 2,
+    },
+    {
+      id: 3,
+      value: 3,
+    },
+    {
+      id: 4,
+      value: 4,
+    },
+    {
+      id: 5,
+      value: 5,
+    },
+  ];
+  const initialState = {
+    shortName: "",
+    fullName: "",
+    text: "",
+    risk: 1,
+    licenseUrl: "",
+    isCandidate: true,
+    mergeRequest: false,
+  };
```

Comment:
@GMishx it is getting its `isCandidate` value from here which is true.

Reply:
In case, someone else is calling the `createCandidateLicenseApi()` function, they will have to make sure the `isCandidate: true` is set. But that should be the behavior of a generic function to create any kind of licenses, not one which is designed specifically for candidate licenses.

Reply:
Okay so I'll hardcoat in the api function itself instead in the component.

Reply:
@GMishx I have fixed it.